application.md

```diff
@@ -1,5 +1,141 @@
+administrative overhead.
+Both the SunPy and Astropy projects have participated in GSoC from 2013 to 2015
+under the Python Software Foundation umbrella organization, and the experience
+was highly successful for both organisations (a total of X students completed
+their projects during these years, many of them with exceptional performance).
+Additionally the yt project participated last year under NumFocus with positive
+results. 
+We hope that we can replicate and share this experience with other projects as
+part of the OpenAstronomy organization.
+
+## How many potential mentors have agreed to mentor this year?
+
+At the moment we have XX potential mentors for this year’s program (listed on
+our ideas page).
+In order to be selected as mentors, developers need to have made major code
+contributions to their respective project, and be active contributors to date.
+These are therefore 'core' developers that understand the code base very well.
+They must also be in a position to dedicate enough time to properly mentor GSoC
+students.
+A number of the mentors have been GSoC mentors in all the editions since 2013.
+How will you keep mentors engaged with their students? Each sub-organisation has
+weekly teleconferences to discuss theIr development, in these calls they will
+dedicate a time for the GSoC mentors and students to update the progress and
+problems encountered.
+
+/is that engaging enough? A Party or chocolates will have more effect,
+something that will make others to want to mentor in the future/
+
+## How will you help your students stay on schedule to complete their projects?
+
+The students are asked to keep a blog up to date with their progress.
+Additionally, from the timeline in their application and with the help of the
+mentors a number of milestones will be set.
+All this information will be publicly available.
+In case of problems or delays most of the mentors have experience in help the
+students to identify the causes to drag them behind.
+The organisation admins will keep a close eye on the projects with inexperienced
+mentors to be able to help before to late.
+If after all the efforts the student don't show interest or disappears, the
+student will be failed.
+
+## How will you get your students involved in your community during GSoC?
+
+Working together with the community is a very important aspect for all of the
+sub-organisations under OpenAstronomy.
+We have created a page with guidelines [1] on what students can do to increase
+their chances to be accepted into the OpenAstronomy GSoC programme.
+To ensure students interact with our communities before their applications are
+submitted, one of the requirements of applications will be that students must
+submit at least one patch as part of their application, which will necessarily
+require that the students interact with the existing developer community.
+Once we select students, we will encourage them to further discuss their project
```

Comment:
"encourage"->"require"?

 And maybe give specific examples of *how* they do this.  Maybe something like:

... with the community, via the individual communities' main communication methods (e.g. the astropy astropy-dev and sunpy-dev Google Groups) before the coding period ...